<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingleItemWidget</class>
 <widget class="QWidget" name="SingleItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>74</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Item</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>74</y>
     <width>571</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid rgba(255,255,255,1);
color: rgb(255, 255, 255);
opacity:0.08;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_ItemDate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>51</y>
     <width>96</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Item">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>300</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>20</y>
     <width>34</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
